src/containers/Output.js

```diff
@@ -175,32 +172,35 @@ class Output extends Component {
   getSize() {
     const { game, songs, chains, phrases, sprites, map } = this.props
 
-    const gameText = assembleOrderedGame(game)
+    function getTokenCount(src) {
```

Comment:
Could you put the `getTokenCount` function outside `getSize()`? That way we save the overhead of function creation.

Reply:
Yup on it. I will move both of the inline functions

Reply:
Done. I inlined renderCodeCount as well cause I think it is simpler that way.